### ISSUE TITLE
Better Canadian Postcode Data

### ISSUE THREAD

**Issue by gelsas** [openaddresses/openaddresses#5650]
I noticed that the Canadian Postcodes are not really good in openaddresses yet.

This company is making monthly updated canadian zip code data (890.000 entries right now) avaiable for free: https://www.serviceobjects.com/blog/unique-us-canadian-zip-code-files-available-download/
https://www.serviceobjects.com/public/zipcode/
ZipCodeFiles.zip

This would make Canadian address data much much more accurate.

**Comment by nvkelso** [comment]
https://www.serviceobjects.com/zip-eula says:

> "You may not (nor allow any third party) to, resell, incorporate Zip File into in any product or services, copy, distribute, sell, disclose, lend, transfer, convey, modify, decompile, disassemble or reverse engineer Zip File for any purpose whatsoever."

So no dice.

**Comment by ingalls** [comment]
@gelsas I would love to get better canada post data into the project. Unfortunately Stats Can only provides the FSA polygons, and Canada Post claims copyright on the full FSA/LDU portion of the postcode (FSA = first 3 chars, LDU = last 3). We have a couple of Canadian sources that provide postal information but the vast majority do not for this reason.

They also have a nasty habit of suing those who try to free the postcode: https://geocoder.ca/?sued=1

So basically our hands are tied until Canada Post realises that as a crown corp. they should be subject to the same data standards as the federal gov't (unlikely) or someone sues them for access (also unlikely)
